### PR TITLE
[dv/clkmgr] Update V1 checklist

### PR DESCRIPTION
**Pull request by matutem**
Signed-off-by: Guillermo Maturana <maturana@google.com>

### CODE REVIEW

File: hw/ip/clkmgr/doc/checklist.md [hunk 1]
```diff
@@ -115,16 +115,16 @@ Review        | Signoff date            | Not Started |
...
-Testbench     | [TB_TOP_CREATED][]                    | Not Started |
-Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Not Started |
-Testbench     | [SIM_TB_ENV_CREATED][]                | Not Started |
-Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | Not Started |
-Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Not Started |
-Testbench     | [TB_GEN_AUTOMATED][]                  | Not Started |
-Tests         | [SIM_SMOKE_TEST_PASSING][]            | Not Started |
-Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Not Started |
+Documentation | [DV_DOC_DRAFT_COMPLETED][]            | Done        | [CLKMGR DV document]()
+Documentation | [TESTPLAN_COMPLETED][]                | Done        | [CLKMGR Testplan]()
```
Comment (sriyerg) [inline hw/ip/clkmgr/doc/checklist.md:119]: The links appear to be missing.

Comment (matutem) [reply hw/ip/clkmgr/doc/checklist.md]: Done.

File: hw/ip/clkmgr/doc/checklist.md [hunk 2]
```diff
@@ -115,16 +115,16 @@ Review        | Signoff date            | Not Started |
...
+Documentation | [TESTPLAN_COMPLETED][]                | Done        | [CLKMGR Testplan]({{<relref "dv/index.md#dv-plan" >}})
+Testbench     | [TB_TOP_CREATED][]                    | Done        |
+Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
+Testbench     | [SIM_TB_ENV_CREATED][]                | Done        |
+Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | Done        |
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
+Testbench     | [TB_GEN_AUTOMATED][]                  | Done        |
+Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
+Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Done        |
 Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | Not Started |
```
[the remark this answers is not in the document]
Comment (matutem) [reply hw/ip/clkmgr/doc/checklist.md:128]: I updated some. Not sure about the others.

File: hw/ip/clkmgr/doc/checklist.md [hunk 3]
```diff
@@ -115,16 +115,16 @@ Review        | Signoff date            | Not Started |
...
+Documentation | [DV_DOC_DRAFT_COMPLETED][]            | Done        | [CLKMGR DV document]({{<relref "dv" >}})
+Documentation | [TESTPLAN_COMPLETED][]                | Done        | [CLKMGR Testplan]({{<relref "dv/index.md#dv-plan" >}})
+Testbench     | [TB_TOP_CREATED][]                    | Done        |
+Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
+Testbench     | [SIM_TB_ENV_CREATED][]                | Done        |
+Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | Done        |
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
+Testbench     | [TB_GEN_AUTOMATED][]                  | Done        |
+Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
+Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Done        |
```
[the remark this answers is not in the document]
Comment (matutem) [reply hw/ip/clkmgr/doc/checklist.md]: Done

File: hw/ip/clkmgr/doc/checklist.md [hunk 4]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
 Regression    | [SIM_SMOKE_REGRESSION_SETUP][]        | Not Started |
 Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | Not Started |
 Regression    | [FPV_REGRESSION_SETUP][]              | Not Started |
-Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Not Started |
-Code Quality  | [TB_LINT_SETUP][]                     | Not Started |
+Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Done        |
+Code Quality  | [TB_LINT_SETUP][]                     | Done        |
 Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | Not Started |
 Review        | [DESIGN_SPEC_REVIEWED][]              | Not Started |
 Review        | [TESTPLAN_REVIEWED][]                 | Not Started |
```
Comment (sriyerg) [inline hw/ip/clkmgr/doc/checklist.md:137]: This would be marked as done after the review meeting.

File: hw/ip/clkmgr/doc/checklist.md [hunk 5]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
 Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Not Started | Exception (?)
```
Comment (sriyerg) [inline hw/ip/clkmgr/doc/checklist.md:138]: See our DV methodology doc for this - just a reminder to ensure you have thought through all those specific scenarios that DV engineers tend to overlook.

File: hw/ip/clkmgr/doc/checklist.md [hunk 6]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
+Testbench     | [TB_GEN_AUTOMATED][]                  | Done        |
+Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
+Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | N.A.        |
 Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | Not Started |
 Tool Setup    | [SIM_ALT_TOOL_SETUP][]                | Not Started |
 Regression    | [SIM_SMOKE_REGRESSION_SETUP][]        | Not Started |
 Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | Not Started |
 Regression    | [FPV_REGRESSION_SETUP][]              | Not Started |
-Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Not Started |
```
[the remark this answers is not in the document]
Comment (matutem) [reply hw/ip/clkmgr/doc/checklist.md:133]: It is missing one more covergroup for extclk, but all others are in place.

File: hw/ip/clkmgr/doc/checklist.md (hunk 4 above)
Comment (matutem) [reply hw/ip/clkmgr/doc/checklist.md:137]: Okay.

File: hw/ip/clkmgr/doc/checklist.md [hunk 7]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
 Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | Not Started |
-Tool Setup    | [SIM_ALT_TOOL_SETUP][]                | Not Started |
-Regression    | [SIM_SMOKE_REGRESSION_SETUP][]        | Not Started |
-Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | Not Started |
-Regression    | [FPV_REGRESSION_SETUP][]              | Not Started |
-Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Not Started |
-Code Quality  | [TB_LINT_SETUP][]                     | Not Started |
-Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | Not Started |
-Review        | [DESIGN_SPEC_REVIEWED][]              | Not Started |
+Tool Setup    | [SIM_ALT_TOOL_SETUP][]                | Done        | xcelium: 49/50 smoke test runs pass
```
Comment (sriyerg) [inline hw/ip/clkmgr/doc/checklist.md:129]: Nit: i think Just specifying Xcelium is sufficient - if you provide the statistic, somebody is bound to ask more questions about that one failing seed :)

File: hw/ip/clkmgr/doc/checklist.md [hunk 8]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
+Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | N.A.        |
+Review        | [DESIGN_SPEC_REVIEWED][]              | Done        |
 Review        | [TESTPLAN_REVIEWED][]                 | Not Started |
-Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Not Started | Exception (?)
-Review        | [V2_CHECKLIST_SCOPED][]               | Not Started |
+Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Done        | Exception (?)
+Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
 
 [DV_DOC_DRAFT_COMPLETED]:             {{<relref "/doc/project/checklist.md#dv_doc_draft_completed" >}}
 [TESTPLAN_COMPLETED]:                  {{<relref "/doc/project/checklist.md#testplan_completed" >}}
```
Comment (sriyerg) [inline hw/ip/clkmgr/doc/checklist.md:142]: nit: spacing is off

File: hw/ip/clkmgr/doc/checklist.md [hunk 9]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
+Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | Done        |
+Regression    | [FPV_REGRESSION_SETUP][]              | N.A.        |
+Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Done        |
+Code Quality  | [TB_LINT_SETUP][]                     | Done        |
+Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | N.A.        |
+Review        | [DESIGN_SPEC_REVIEWED][]              | Done        |
 Review        | [TESTPLAN_REVIEWED][]                 | Not Started |
-Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Not Started | Exception (?)
-Review        | [V2_CHECKLIST_SCOPED][]               | Not Started |
+Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Done        | Exception (?)
```
Comment (sriyerg) [inline hw/ip/clkmgr/doc/checklist.md:138]: Perhaps remove the note, or indicate the exceptions (clkmgr probably does not have debug or performance usecases to verify).

File: hw/ip/clkmgr/doc/checklist.md [hunk 10]
```diff
@@ -115,31 +115,31 @@ Review        | Signoff date            | Not Started |
...
+Documentation | [TESTPLAN_COMPLETED][]                | Done        | [CLKMGR Testplan]({{<relref "dv/index.md#dv-plan" >}})
+Testbench     | [TB_TOP_CREATED][]                    | Done        |
+Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
+Testbench     | [SIM_TB_ENV_CREATED][]                | Done        |
+Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | Done        |
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
+Testbench     | [TB_GEN_AUTOMATED][]                  | Done        |
+Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
+Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | N.A.        |
 Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | Not Started |
```
Comment (sriyerg) [inline hw/ip/clkmgr/doc/checklist.md:128]: This would also be N.A.

Comment (sriyerg) [reply hw/ip/clkmgr/doc/checklist.md:128]: FYI, you seen to have missed this one.

**Review by sriyerg — APPROVED** [review]
LGTM, thanks Matute!

**Comment by sriyerg** [comment]
Please update `hw/ip/clkmgr/data/clkmgr.prj.hjson` to reflect the V1 status in a separate PR.

**Review by weicaiyang — APPROVED** [review]
LGTM

**Pull request merged**